package.json

```diff
@@ -25,14 +25,16 @@
   },
   "homepage": "https://github.com/brizental/gleanjs#readme",
   "devDependencies": {
+    "@babel/core": "^7.11.6",
```

Comment:
It seems like jest required this all of sudden. Not sure what happened here, will investigate.

Reply:
Won't investigate as we won't go ahead with jest beyond this proof of concept and will instead use mocha. This is due to Windows issues, please see more info at: https://github.com/facebook/jest/issues/7631